src/components/stats/input.js

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+import {
+  unstable_LowPriority as LowPriority,
+  unstable_runWithPriority as runWithPriority,
+  unstable_scheduleCallback as scheduleCallback
+} from "scheduler";
+
+const InputContainer = styled.p`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+const InputTag = styled.input`
+  font-size: 24px;
+  border: 2px solid black;
+  border-radius: 3px;
+  width: 300px;
+  text-align: center;
+  margin: 10px;
+`;
+
+const Input = ({ async, onChange }) => {
+  const [count, setCount] = useState(10);
+
+  useEffect(() => onChange(count), []);
+
+  const changeHandler = event => {
+    const newCount = parseInt(event.target.value, 10);
+    setCount(newCount);
+
+    if (async) {
+      runWithPriority(LowPriority, () => {
```

Comment:
AAAW MEEEEN ! THIS IS CRAZY ! When did it come out?

Reply:
Not sure in which version. But I discover the API with this article: https://philippspiess.com/scheduling-in-react/